modules/upcloud_tag.py

```diff
@@ -166,44 +169,44 @@ def run(module, tag_manager):
         (don't remove them from upcloud's available tags, might be present in other servers)
     """
 
-    state =         module.params['state']
-    tags =          module.params['tags']
-    uuid =          module.params.get('uuid')
-    hostname =      module.params.get('hostname')
-    ip_address =    module.params.get('ip_address')
+    state = module.params["state"]
+    tags = module.params["tags"]
+    uuid = module.params.get("uuid")
+    hostname = module.params.get("hostname")
+    ip_address = module.params.get("ip_address")
 
     changed = False
 
     if not uuid:
         if hostname:
-            uuid = firewall_manager.determine_server_uuid_by_hostname(hostname)
+            uuid = tag_manager.determine_server_uuid_by_hostname(hostname)
```

Comment:
```suggestion
            uuid = tag_manager.get_server_uuid_by_hostname(hostname)
```

Reply:
I think this is a better naming for this function

Reply:
Since we will be deprecating this repo I’d not make too many lifecycle management changes.